ecosystem/typescript/sdk/src/aptos_types/type_tag.ts

```diff
@@ -407,3 +406,10 @@ export class TypeTagParser {
     throw new Error("Invalid type tag.");
```

Comment:
lets keep it consistent
```suggestion
    bail("Invalid type tag.");
```